docs/bayes.md

```diff
@@ -135,6 +135,77 @@ classifier.train("Cat", "I can has cat")
 classifier.train("Dog", "I don't always bark at night")
 ```
 
+## Custom Tokenizer
+
+By default the classifier tokenizes given inputs as a white-space separeted terms.
+If you want to use different tokenizer, give it via the `:tokenizer` option.
+Tokenizer must be an object having a method named `call`, or a lambda.
+The function must return tokens as instances of `ClassifierReborn::Tokenizer::Token`.
+
+```ruby
+require 'classifier-reborn'
+
+module BigramTokenizer
+  module_function
+  def call(str)
+    str.each_char
+       .each_cons(2)
+       .map do |chars| ClassifierReborn::Tokenizer::Token.new(chars.join) end
+  end
+end
+
+classifier = ClassifierReborn::Bayes.new tokenizer: BigramTokenizer
+```
+
+or
+
+```ruby
+require 'classifier-reborn'
+
+bigram_tokenizer = lambda do |str|
+  str.each_char
+     .each_cons(2)
+     .map do |chars| ClassifierReborn::Tokenizer::Token.new(chars.join) end
+end
+
+classifier = ClassifierReborn::Bayes.new tokenizer: bigram_tokenizer
+```
+
+## Custom Token Filters
+
+By default classifier rejects stopwords from tokens.
+This behavior is implemented based on filters for tokens.
+If you want to use more token filters, give them via the `:token_filter` option.
+A filter must be an object having a method named `call`, or a lambda.
+
+```ruby
+require 'classifier-reborn'
+
+module CatFilter
+  module_function
+  def call(tokens)
+    tokens.reject do |token|
+      /cat/i === token
+    end
+  end
+end
+
+white_filter = lambda do |tokens|
+  tokens.reject do |token|
+    /white/i === token
+  end
+done
+
+filters = [
+  CatFilter,
+  white_filter
```

Comment:
@piroor, are we missing a comma at the end here? Also, the last element of the array has an unnecessary comma.